domains/itzwebster.json

```
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "ItzWebster",
+           "email": "webstergamingyt@gmail.com",
+           "discord": "481068576363773972"
+        },
+    
+        "record": {
+            "CNAME": "https://oliverbot.xyz"
```

```suggestion
            "CNAME": "oliverbot.xyz"
```
Also please add more stuff to your website